frontend/ui/app/[locale]/home/(about)/AboutSection.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useTranslations } from "next-intl";
+import styles from "./AboutSection.module.css";
+import Image from "next/image";
+import DesignIcon from "./DesignIcon";
+import SafetyIcon  from "./SafetyIcon";
+import CalmIcon from "./CalmIcon";
+
+const AboutSection = () => {
+  const t = useTranslations();
+
+  return (
+    <section>
+      <div className={styles.aboutContainer}>
+        <section>
+          <h1>{t("Index.about.about_heading")}</h1>
+          <div className={styles.textFormating}>{t("Index.about.about_description")}</div>
+        </section>
+        <section>
+          <Image src={"/assets/cat_about.png"} alt={"White cat"}
```

Comment:
alt could be translated as well. Different languages, different needs for accessibility